client/templates/includes/editCustomStoichSpecie.pug

```diff
@@ -1,10 +1,10 @@
 div
   div.inline
-    button.btn.btn-outline-secondary(class='btn-sm', data-hook='decrement') -
-  span.custom(data-hook="ratio")
+    button.btn.btn-outline-secondary(class='btn-sm', data-hook='decrement') -id="", 
```

Comment:
This id attribute should be inside the parens, the '-' is the text for the button